lib/src/cli.rs

```diff
@@ -6,6 +6,7 @@
 //! such as `rpm-ostree` can directly reuse it.
 
 use anyhow::Result;
+use cap_std_ext::cap_std;
```

Comment:
I think you'd want `ostree::cap_std` here, in order to match what `open_at_dir` wants.

At the time of new releases, `cap_std_ext::cap_std` and `ostree::cap_std` may be at incompatible versions.

Reply:
You're definitely right in this instance, but in general we use both `cap-std-ext` and `ostree`, and I don't think we really want to have two different versions of cap-std in the dependency graph.

IOW semver bumps there are going to involve churn for us no matter what I think.